types/index.d.ts

```diff
@@ -289,10 +289,61 @@ declare namespace PhonegapPluginPush {
 		from?: string
 		notId?: string
 	}
-
+  /**
+   * Set Channel Variable,once created cannot be changed programmtically for importance and vibration
+   */
+  interface channel{
+    /**
+     * channel ID
+     */
+    id: string
+    /**
+     * camillebeaumont name and description fix
+     */
+    name: string 
+    /**
+     * camillebeaumont name and description fix,channel description
+     */
+    description: string
+    /**
+     * channel importance level
+     */
+    importance: integer
+    /**
+     * enable vibration for the notification
+     */
+    vibration: boolean
+    /**
+     * Set notitfication visibility 
+     */
+    visibility: integer
+    /**
+     * Set Channel Group
```

Comment:
Let's use the same docs from the API.md table:
```suggestion
     * Sets channel group id. Assign it to a Channel Group Created. Cannot be changed once set.
```